sample.env

```diff
@@ -0,0 +1,9 @@
+AZUREKEY= azure_key
+SPOTIFY_API_ID= api_id
+SPOTIFY_CLIENT_SECRET= secret
```

Comment:
Do you also wanna edit the README for the env and add to the 'Built With' for the technologies?

Reply:
Ah yup sure I'll do go that!